src/Majora/Framework/Loader/Bridge/Doctrine/DoctrineLoaderTrait.php

```diff
@@ -88,21 +82,61 @@ protected function toEntityCollection($result)
     }
 
     /**
-     * Loads data from repository
-     * then cast it to proper classes if not.
+     * Create entity query.
+     * Proxy to base query builder method to use to custom all queries from this loader.
+     *
+     * @param string $alias
      *
+     * @return QueryBuilder
+     */
+    protected function createQuery($alias = 'entity')
+    {
+        return $this->entityRepository->createQueryBuilder($alias);
+    }
+
+    /**
+     * create query an filter it with given data.
+     *
+     * @param array $filters
+     *
+     * @return Query
+     */
+    private function createFilteredQuery(array $filters)
+    {
+        $qb = $this->createQuery('entity');
+
+        foreach ($filters as $field => $filter) {
+            $qb->andWhere(is_array($filter) ?
+                    sprintf('entity.%s in (:%s)', $field, $field) :
+                    sprintf('entity.%s = :%s', $field, $field)
+                )
```

Comment:
this syntax is really hard to read
I'd prefer that you place `?` and `:` at the beginning of each line, such as:
```
            $qb->andWhere(is_array($filter)
                ? sprintf('entity.%s in (:%s)', $field, $field)
                : sprintf('entity.%s = :%s', $field, $field)
            )
```